src/components/graph/MetaGraph.ts

```diff
@@ -185,19 +188,22 @@ export class MetaGraph {
 
     updateGraph(metaNodeModel: MetaNodeModel, cursorX: number, cursorY: number) {
         // update the graph for right parent children relationship
+        let pathUpdated = false;
         this.updateNodeContainerBoundingBox(metaNodeModel);
         if (!this.parentUpdating) {
             this.parentUpdating = true;
             let parent: MetaNodeModel|undefined = this.rootContainsNode(metaNodeModel, cursorX, cursorY);
             let newPath = this.findNewPath(metaNodeModel, parent, cursorX, cursorY);
```

Comment:
@ddelpiano `const` ?

Reply:
nope, later I am reassigning this if the path changes, const would not work.